### PR TITLE
fix: crash after updating from scala to 4.1.3

### PR DESCRIPTION
**Pull request by MohamadJaara**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

crash after updateing to 4.1.3

### Solutions

obscure sql statment

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by MohamadJaara** [comment]
after a suggestion form @vitorhugods i added a work around for the [sqlDelight verification bug](https://github.com/cashapp/sqldelight/issues/4154) by deleting the 2.db and all the migrations form 1 -> 33.sqm

File: persistence/src/androidMain/kotlin/com/wire/kalium/persistence/dao/message/MessageExtensions.kt
Comment (vitorhugods) [inline persistence/src/androidMain/kotlin/com/wire/kalium/persistence/dao/message/MessageExtensions.kt:1]: This is probably something we can remove completely now that's Multiplatform and supported by SQLDelight.

This whole `MessageExtensions` thing.

Not that this needs to be done in this PR, just saying it out loud for now.

**Review by vitorhugods — APPROVED** [review]
👨‍🍳🤌🏻

File: scripts/dev.mk [hunk 1]
```diff
@@ -4,9 +4,19 @@ db/verify-global-migration:
 	./gradlew :persistence:verifyCommonMainGlobalDatabaseMigration
 
 db/verify-user-migration:
+	rm persistence/src/commonMain/db_user/schemas/2.db
+	for i in {1..33}; \
+	do \
+		rm persistence/src/commonMain/db_user/migrations/$$i.sqm; \
+	done
```
Comment (vitorhugods) [inline scripts/dev.mk:11]: Could you please just add a comment here saying why this is here. So if we ever forget and someone else finds it 😅

**Pull request merged**